keyboards/macroboard/config.h

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2021 Michał Szczepaniak
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0xFEED
+#define PRODUCT_ID   0x6465
+#define DEVICE_VER   0x0001
+#define MANUFACTURER QMK
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 6
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { A15, B3, B4, B5, B7  }
+#define MATRIX_COL_PINS { B12, B13, B14, B15, A8, A10 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+#define ADC_PIN A0
```

Comment:
Is this actually being used?

Reply:
I don't think so at least i'm not using them consciously. I was taking inspiration from some other code, i can remove if needed

Reply:
If it's not being used, then, yeah, should remove.

Reply:
It fails to build if i remove diode direction so i'm going to leave that one in, and i'm using diodes in handwiring

Reply:
> and i'm using diodes in handwiring

Then this should be relocated to within the `handwired` folder